examples/plotting/customjs_expr.py

```diff
@@ -1,3 +1,10 @@
+''' This example demonstrates using CustomJS Expression to create a sinusoidal line chart with Bokeh.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.models.CustomJSExpr, bokeh.model.DataModel
+    :refs: :ref:`ug_interaction_js_callback`
+    :keywords: CustomJS, CustomJSExpr
```

Comment:
we generally avoid re-iterating the APIs in keywords
```suggestion
    :keywords: js callbacks, expressions
    
```
also a blank line after metadata is typical